spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/ConfigCommands.java

```diff
@@ -270,17 +296,27 @@ public void afterPropertiesSet() throws Exception {
 		//Only invoke this lifecycle method if the shell is executing in stand-alone mode.
 		if (applicationContext != null && !applicationContext.containsBean("streamDefinitionRepository")) {
 			initialized = true;
-			target(this.serverUri, this.userName, this.password);
+			target(this.serverUri, this.userName, this.password, isSkipSslValidation(this.environment));
 		}
 	}
 
-	@Bean
-	public static RestTemplate restTemplate() {
-		return DataFlowTemplate.getDefaultDataflowRestTemplate();
+	private boolean isSkipSslValidation(Environment env) {
```

Comment:
Depending on answer to my comment for line 290, this could be removed altogether